src/components/TableModule/TableModuleCell.test.tsx

```diff
@@ -86,7 +86,7 @@ test('it renders with "isLastCellInRow"', async () => {
     </RenderInTable>
   );
   const root = await findByTestId(testIds.bodyCell);
-  expect(root).toHaveClass('ChromaTableModule-tableRowCellAlignRight');
+  expect(root).toHaveClass('ChromaTableModule-tableRowCell');
```

Comment:
Ah interesting.  Yeah this test should've caught this; however, it wasn't written correctly 🙈 .  My b.